app/interactions/users/authenticate_user.rb

```diff
@@ -5,11 +5,6 @@ class AuthenticateUser < ActiveInteraction::Base
   string :token
 
   def execute
-    user = User.find_or_initialize_by(email: email)
-    user.first_name = first_name
-    user.last_name = last_name
-    user.token = token
-    user.save
-    user
+    User.find_by(email: email) || User.create(inputs)
```

Comment:
That was awswered [with code](https://github.com/got-board-team/got-board-api/pull/11/commits/2337b2b29fc954cb26e325819ecf9274549d957e) :wink: